modules/getting-started-cli-creating-secret.adoc

```diff
@@ -1,53 +1,41 @@
 // Module included in the following assemblies:
 //
-// * getting-started/openshift-cli.adoc
+// * tutorials/dev-app-cli.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="getting-started-cli-creating-secret_{context}"]
+= Providing access to the database by creating a secret
 
-= Creating a secret
+The `nationalparks` application needs information, such as the database name, username, and passwords, on how to access the MongoDB database. However, because this is sensitive information, it should not be stored directly in the pod.
```

Comment:
```suggestion
The `nationalparks` application needs information, such as the database name, username, and passwords, to access the MongoDB database. However, because this information is sensitive, you should not store it directly in the pod.
```